### PR TITLE
Survey Sliders plugin

### PR DESCRIPTION
**Pull request by Max-Lovell**
This plugin adds sliders to questionnaires, i.e. multiple on a page. Done on behalf of Dominique Makowski, relevant issue was raised previously: https://github.com/jspsych/jspsych-contrib/issues/94

### CODE REVIEW

File: .changeset/clean-seals-behave.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@jspsych-contrib/plugin-survey-slider": patch
```
Comment (jodeleeuw) [inline .changeset/clean-seals-behave.md:2]: this should probably be major instead of patch, unless you don't want this to be marked as version 1.0.0

**Comment by jodeleeuw** [comment]
Thanks @Max-Lovell !

**Comment by jodeleeuw** [comment]
@Max-Lovell it also looks like package-lock.json is out of sync. Try running npm i to update and push the updated package-lock.json file.

File: .changeset/clean-seals-behave.md (hunk 1 above)
Comment (Max-Lovell) [reply .changeset/clean-seals-behave.md:2]: Okay then sure, pushed that now

**Comment by jodeleeuw** [comment]
All set! Thanks again @Max-Lovell !

**Pull request merged**